toolset/setup/linux/languages/pypy.sh

```diff
@@ -6,15 +6,14 @@ RETCODE=$(fw_exists ${IROOT}/pypy.installed)
   return 0; }
   
 PYPY_ROOT=$IROOT/pypy
+PYPY_VERSION=5.0.0
 
-fw_get -o pypy-4.0.1-linux64.tar.bz2 https://bitbucket.org/pypy/pypy/downloads/pypy-4.0.1-linux64.tar.bz2
-fw_untar pypy-4.0.1-linux64.tar.bz2
-ln -sf pypy-4.0.1-linux64 pypy
+fw_get -o pypy-${PYPY_VERSION}-linux64.tar.bz2 https://bitbucket.org/pypy/pypy/downloads/pypy-${PYPY_VERSION}-linux64.tar.bz2
+fw_untar pypy-${PYPY_VERSION}-linux64.tar.bz2
+mv pypy-${PYPY_VERSION}-linux64 pypy
 
-fw_get -o get-pip.py https://bootstrap.pypa.io/get-pip.py
-
-# Ensure pip is installed
-$PYPY_ROOT/bin/pypy get-pip.py
+$PYPY_ROOT/bin/pypy -m ensurepip
+$PYPY_ROOT/bin/pip -U pip setuptools wheel
```

Comment:
The script seems to be failing here. I believe the line should be:

    $PYPY_ROOT/bin/pip install -U pip setuptools wheel